services/validators_test.go

```diff
@@ -43,6 +44,38 @@ func TestValidateJob(t *testing.T) {
 	}
 }
 
+func TestValidateBridgeType(t *testing.T) {
+	t.Parallel()
+
+	store, cleanup := cltest.NewStore()
+	defer cleanup()
+
+	tt := models.BridgeType{}
+	tt.Name = "solargridreporting"
+	u, err := url.Parse("https://denergy.eth")
+	assert.Nil(t, err)
+	tt.URL = models.WebURL{u}
+	assert.Nil(t, store.Save(&tt))
+
+	tests := []struct {
+		description string
+		name  string
+		want  error
+	}{
+		{"existing external adaptor", "solargridreporting",
+		errors.New("bridge type validation: adaptor solargridreporting exists")},
+		{"existing core adaptor", "ethtx",
+		errors.New("bridge type validation: adaptor ethtx exists")},
+		{"new external adaptor", "gdaxprice", nil},
+	}
+
+	for _, test := range tests {
+		bt := &models.BridgeType{Name: test.name}
```

Comment:
Please [wrap this inner code](https://github.com/smartcontractkit/chainlink/blob/master/adapters/adapter_test.go#L42-L53) in a `t.Run`. It gives nicer output, and allows you to target specific test cases.